react-lists/my-app/src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import { PokemonList } from './PokemonList';
+import './App.css';
+
+export default function App() {
+  type Props = [
```

Comment:
You should declare this type a little differently. The way you've declared it, your list must have exactly 5 elements. To make it more general so you can have any number of elements, you'd declare it as an array, like this:

```
type Props = {
  number: string;
  name: string;
}[];
```

The best way to do it, though, since you might at some point want to refer to the things inside the array, is to have a separate type for the element itself:

```
type Pokemon = {
  number: string;
  name: string;
};
type Props = Pokemon[];
```